src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -64,7 +66,15 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     String password = request.getParameter("password");
     String hashedPassword = BCrypt.hashpw(password, BCrypt.gensalt());
 
-    User user = new User(UUID.randomUUID(), username, hashedPassword, Instant.now());
+    List <String> validAdmins = new ArrayList<>();
```

Comment:
We can use the Lists class do all this initialization in one line: http://www.baeldung.com/guava-lists

Reply:
So… I also include the library of guava?

Reply:
Well. You would include `com.google.common.collect.Lists` how I found this is by looking up "Lists java" on Google and finding this page: https://google.github.io/guava/releases/23.0/api/docs/com/google/common/collect/Lists.html. And this page shows what the import is a the top.

Reply:
But I will have to add Guava as a dependecy for using this

Reply:
Ahh I guess you are right. Lmk if you need help with that.

Reply:
Don't worry I leaved the arrays behind